data/remnant missions.txt

```diff
@@ -139,6 +139,99 @@ mission "Remnant Blood Test Patch"
 
 
 
+mission "lost in ember waste"
+	landing
+	invisible
+	to offer
+		has "remnant blood test pure"
+		not "license: Remnant"
```

Comment:
I don't see a requirement for either this mission or its followup to actually be given *in* Remnant space, which means that visiting just long enough for the blood test, leaving, then selling your keystone could result in a very strange conversation about being "stuck in this strange region of space" when you're in Republic territory.

Reply:
Always gotta be something missing, I guess.